runtime/test-fw/src/cert_tests.rs

```diff
@@ -0,0 +1,47 @@
+// Licensed under the Apache-2.0 license
+
+//! Send DICE certificates over the mailbox
+
+#![no_main]
+#![no_std]
+
+use caliptra_registers::mbox::enums::MboxStatusE;
+use caliptra_runtime::{dice, mailbox::Mailbox};
+use caliptra_test_harness::{runtime_handlers, test_suite};
+
+fn mbox_responder() {
+    let soc_ifc = caliptra_registers::soc_ifc::RegisterBlock::soc_ifc_reg();
+    soc_ifc.cptra_flow_status().write(|w| w.ready_for_fw(true));
```

Comment:
Could this be `soc_ifc.flow_status_set_ready_for_firmware()`? But also, isn't that already done by ROM?

Reply:
Oh right. Removed.